src/validations/css/prettycss.js

```diff
@@ -5,9 +5,34 @@ import endsWith from 'lodash/endsWith';
 
 const RADIAL_GRADIENT_EXPR =
   /^(?:(?:-(?:ms|moz|o|webkit)-)?radial-gradient|-webkit-gradient)/;
+
+const FILTER_VALUE_EXPR =
+  new RegExp([
+    'blur\\(',
+    'brightness\\(',
+    'contrast\\(',
+    'drop-shadow\\(',
+    'grayscale\\(',
+    'hue-rotate\\(',
+    'invert\\(',
+    'opacity\\(',
+    'saturate\\(',
+    'sepia\\(',
+    'inherit',
+    'initial',
+    'unset'].join('|'));
```

Comment:
Also—`PrettyCSS` generally doesn’t support `initial` and `unset`; and I don’t think anyone ever uses them, so I would say let’s drop them here as well